packages/web-client/app/templates/card-pay/payments.hbs

```diff
@@ -69,11 +69,7 @@
       </header>
       <section class="card-pay-dashboard__payments-cards">
         {{#each this.merchantSafes as |merchantSafe|}}
-          <CardPay::MerchantSafe
-            @safe={{merchantSafe}}
-            @waitForCustomization={{true}}
-            data-test-merchant-dashboard-card
-          />
+          <CardPay::SafeBalances @safe={{merchantSafe}} />
```

Comment:
This component is now used in both /payments and /deposit-withdrawal routes. 

Now that this component contains the safe info, together with balances, would it make sense to rename it to `CardPay::Safe` (and all the CSS that goes with it)?

Reply:
I like that idea!